packages/client/hmi-client/src/components/sidebar-panel/simulation-plan-sidebar-panel.vue

```diff
@@ -7,7 +7,7 @@ defineProps<{
 }>();
 
 const router = useRouter();
-const openSimulationPlan = (runId: string) => {
+const openSimulationPlan = (runId: string | number) => {
```

Comment:
Unfortunately, this is not the case at the backend since the backend is NOT applying any mapping at the moment, so we receive the raw data from the data service and it is coming as number in this case.
That is something that we should review after Dec 8 to properly ensure applying type mapping at the backend which will simplify much of this code!

Reply:
@dgauldie @dvince2 ⬆️